### PR TITLE
Rename warning

### PR DESCRIPTION
**Pull request by htorrence**
1) add warning about name change to final release under "apparate" name
2) add github actions to run CI checks + push to PyPi

Pull Request Checklist
 - [x] Pull request includes a description of the change and the reason behind it
 - [x] CI checks pass
 - [x] CHANGELOG has been updated
 - [x] Version in `_version.py` has been updated
 - [x] README and docs have been updated (if applicable)

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt -r requirements-dev.txt
+        printf "[DEFAULT]\nhost = https://fake-org.cloud.databricks.com\nprod_folder = /databricks/folder\n" > ~/.apparatecfg
```
Comment (sugi-chan) [inline .github/workflows/pr.yaml:27]: is this print statement supposed to be weird looking? (from a python point of view)

Comment (sugi-chan) [reply .github/workflows/pr.yaml:27]: im guessing yes, but figure I might as well ask lol

File: apparate/cli_commands.py [hunk 2]
```diff
@@ -70,6 +73,8 @@ def upload(path, token, folder):
     """
     The egg that the provided path points to will be uploaded to Databricks.
     """
+    warnings.warn(RENAME_WARNING_MESSAGE)
```
[the remark this answers is not in the document]
Comment (htorrence) [reply apparate/cli_commands.py:76]: yeah, these are pretty much the only commands someone can run

File: .github/workflows/prod.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```
[the remark this answers is not in the document]
Comment (htorrence) [reply .github/workflows/prod.yaml:16]: yeah, it's super nice + easy to put together! These super minor edits to templates github provides

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (htorrence) [reply .github/workflows/pr.yaml:27]: yeah, it's not a python statement, `printf` is a command line utility - this is copied from the current jenkins job and seems to be working fine here.

**Pull request merged**